### ISSUE TITLE
Release patch version

### ISSUE THREAD

**Issue by Jonathas-Conceicao** [malept/crypto-hash#15]
Hello, any idea of when a new minor version can be released? On the recent commits there's a version bump for packages, like hex. I need to reduce binary size on a project I work on that depends on crypto-hash, a new release would work perfectly for us.

**Comment by malept** [comment]
I don't believe that a new minor version is warranted. From what I can tell of the [version diff](https://github.com/KokaKiwi/rust-hex/compare/v0.3.2...v0.4.0), the `hex` crate moved to 2018 edition and added no_std support, both of which should not affect this crate. However, I could release a patch version when I have some time.

**Comment by Jonathas-Conceicao** [comment]
A little confusion of naming conventions on my part there, indeed a patch is what I intended to request. Thanks for the quick response.

**Comment by jyn514** [comment]
I'd like to second the request for a patch version :) Upgrading hex would get rid of a duplicate dependency for cargo (right now it depends on both 0.4 directly and 0.3 through this crate).

**Comment by klensy** [comment]
Hello there, one more year, can you please release minor version? Thanks.

**Comment by klensy** [comment]
Hello, any update?
